<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>myWidget</class>
 <widget class="QWidget" name="myWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>myWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pageSet">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QWidget" name="widget_2" native="true"/>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true"/>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>13</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="begin">
            <property name="text">
             <string>进入游戏</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>18</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="highscore">
            <property name="text">
             <string>高分榜</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>13</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="end">
            <property name="text">
             <string>退出游戏</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>13</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QProgressBar" name="progressBar">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>310</y>
         <width>491</width>
         <height>20</height>
        </rect>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
      <widget class="QLCDNumber" name="lcdNumber">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>70</y>
         <width>64</width>
         <height>23</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
         <stylestrategy>PreferDefault</stylestrategy>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="smallDecimalPoint">
        <bool>false</bool>
       </property>
       <property name="digitCount">
        <number>5</number>
       </property>
       <property name="intValue" stdset="0">
        <number>0</number>
       </property>
      </widget>
      <widget class="QPushButton" name="helper1">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>120</y>
         <width>61</width>
         <height>51</height>
        </rect>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="helper2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>170</y>
         <width>61</width>
         <height>51</height>
        </rect>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="helper3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>220</y>
         <width>61</width>
         <height>51</height>
        </rect>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLCDNumber" name="lcdNumber_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>120</y>
         <width>20</width>
         <height>23</height>
        </rect>
       </property>
       <property name="digitCount">
        <number>1</number>
       </property>
      </widget>
      <widget class="QLCDNumber" name="lcdNumber_3">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>220</y>
         <width>20</width>
         <height>23</height>
        </rect>
       </property>
       <property name="digitCount">
        <number>1</number>
       </property>
      </widget>
      <widget class="QLCDNumber" name="lcdNumber_4">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>170</y>
         <width>20</width>
         <height>23</height>
        </rect>
       </property>
       <property name="digitCount">
        <number>1</number>
       </property>
      </widget>
      <zorder>progressBar</zorder>
      <zorder>lcdNumber</zorder>
      <zorder>helper3</zorder>
      <zorder>helper2</zorder>
      <zorder>helper1</zorder>
      <zorder>lcdNumber_2</zorder>
      <zorder>lcdNumber_3</zorder>
      <zorder>lcdNumber_4</zorder>
     </widget>
     <widget class="QWidget" name="pageOver">
      <widget class="QLabel" name="labelOver">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>110</y>
         <width>191</width>
         <height>131</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="pageWin">
      <widget class="QLabel" name="labelWin">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>70</y>
         <width>151</width>
         <height>151</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page">
      <widget class="QPushButton" name="back">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>280</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>BACK</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowser">
       <property name="geometry">
        <rect>
         <x>405</x>
         <y>40</y>
         <width>101</width>
         <height>192</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>end</sender>
   <signal>released()</signal>
   <receiver>myWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>425</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
